azure-pipelines.yml

```diff
@@ -0,0 +1,16 @@
+trigger:
+  branches:
+    include:
+    - '*'
```

Comment:
do we need it for all branches? currently we do this only for master and version-specific branches, so published SNAPSHOT versions don't intercept.
Github Actions build all commits, but never "publish" anything.